templates/_partial/navbar/_navbar_icons.html.twig

```diff
@@ -2,8 +2,8 @@
 <ul class="navbar-nav nav-icons">
     <!-- GitHub -->
     <li class="nav-item">
-        <a href="{{ global.app.url.github }}" target="_blank" class="nav-link">
-            <i class="fab fa-github"></i>
+        <a href="{{ global.app.url.github_af }}" target="_blank" class="nav-link">
+            <i class="fab fa-github_af"></i>
```

Comment:
You also changed the icon name :stuck_out_tongue:

Reply:
Oh, I thought this has to match as other icons do. Now it makes sense.